molo/core/cookiecutter/scaffold/{{cookiecutter.directory}}/{{cookiecutter.app_name}}/templates/search/search_results.html

```diff
@@ -9,23 +9,29 @@
   </div>
   {% for page in search_results %}
     {% with parent_section=page.get_parent_section ancestor=page.get_parent_section.get_ancestors.last %}
-    {% if ancestor.sectionpage.image %}
       <a href="{% pageurl page %}">
         <div class="nav">
-          <h6>{{ancestor.title}}</h6>
-          <h3>{{page.title}}</h3>
-          <p>{{page.subtitle}}</p>
+          {% if ancestor.sectionpage.image %}
+              <h6>{{ancestor.title}}</h6>
+          {% else %}
+              <h6>{{parent_section.title}}</h6>
+          {% endif %}
+          {% if page.title_highlight %}
+              <h3>{{page.title_highlight|safe}}</h3>
+          {% else %}
+              <h3>{{page.title}}</h3>
+          {% endif %}
+          {% if page.subtitle_highlight or page.body_highlight %}
+              {% if page.subtitle_highlight %}
+                  <p>{{page.subtitle_highlight|safe}}</p>
+              {% elif page.body_highlight %}
+                  <p>{{page.body_highlight|safe}}</p>
```

Comment:
By default, max length is 100 characters. You can increase / decrease it using the `fragment_size` option.

For example:

```python
results = results.highlight(
    fields={
        'title': {'fragment_size': 50},
        'subtitle': {},
        'body': {'fragment_size': 150},
    },
    require_field_match=False
)
```

You can pass [Elasticsearch options](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-request-highlighting.html#_highlighted_fragments) inside braces.

So you can change highlighting tags for `title` like this:

```python
results = results.highlight(
    fields={
        'title': {'pre_tags' : ['<span>'], 'post_tags' : ['</span>']},
        'subtitle': {},
        'body': {},
    },
    require_field_match=False
)
```